.github/workflows/matrix.yml

```
@@ -199,28 +199,28 @@ jobs:
     - name: prepare
       if: ${{ env.SKIP == 'false' }}
       run: bash ./build-scripts/requirements.sh
-    - name: Get Date
-      id: get-date
+    - name: Get ccache vars
+      id: get-vars
       if: ${{ env.SKIP == 'false' }}
       run: |
         echo "::set-output name=date::$(/bin/date -u "+%Y%m%d%H%M")"
```

Actually have a per-day timestamp instead of a per minute one, whoops.
```suggestion
        echo "::set-output name=date::$(/bin/date -u "+%Y%m%d")"
```